src/cryptonote_core/cryptonote_tx_utils.cpp

```diff
@@ -628,7 +634,8 @@ namespace cryptonote
         }
       }
 
-      bool r = construct_tx_with_tx_key(sender_account_keys, subaddresses, sources, destinations, change_addr, extra, tx, unlock_time, tx_key, additional_tx_keys, rct, rct_config, msout);
+      bool shuffle_outs = true;
```

Comment:
Ah, because the default value can't be used with an explicit `use_view_tags`. These function signatures are disturbing, but that's not your fault.